spec/writers/sparql_statement_writer_spec.rb

```diff
@@ -0,0 +1,212 @@
+# frozen_string_literal: true
+
+require 'rialto/etl/writers/sparql_statement_writer'
+require 'rialto/etl/readers/sparql_statement_reader'
+require 'rdf'
+require 'sparql'
+
+RSpec::Matchers.define :have_same_triples do |graph1|
```

Comment:
Typically we'd put stuff like this in `spec/support` so we can use it in more than one test.  Then require it in `spec/spec_helper.rb`

Reply:
@justinlittman Here you go:

* https://github.com/samvera/hyrax/blob/master/spec/spec_helper.rb#L56-L57
* https://github.com/samvera/hyrax/blob/master/spec/support/matchers/response_matchers.rb
* https://github.com/samvera/hyrax/blob/5a9d1be16ee1a9150646384471992b03aab527a5/spec/controllers/hyrax/leases_controller_spec.rb#L56